docs/generators/aspnetcore.md

```diff
@@ -51,6 +51,7 @@ These options may be applied as additional-properties (cli) or configOptions (pl
 |useDefaultRouting|Use default routing for the ASP.NET Core version.| |true|
 |useFrameworkReference|Use frameworkReference for ASP.NET Core 3.0+ and PackageReference ASP.NET Core 2.2 or earlier.| |false|
 |useNewtonsoft|Uses the Newtonsoft JSON library.| |true|
+|useParameterDefaultValues|Use parameter default values in controller methods.| |false|
```

Comment:
thanks for the PR. I don't think we need another option to support default values in the parameters as it should be support by default so I would suggest we remove this option.